query.js

```diff
@@ -627,33 +627,33 @@ function parseExport(res) {
         // pursuits
         if (d.pursuits && d.pursuits.length) d.pursuits.forEach(a => activities.push({
             ...entryBase,
-            type: 'pursuit',
-            details: "",
-            value: a.pursuit,
+            lifeEvents: 'DBITI employment or identifier',
+            eventsDetail1: "",
+            eventsDetail2: a.pursuit,
             place: "",
             startDate: "",
             endDate: "",
             index: ++activityIndex,
         }))
 
-        // occuaptions
+        // occupations
         if (d.occupations && d.occupations.length) d.occupations.forEach(a => activities.push({
             ...entryBase,
-            type: 'occupation',
-            details: a.group,
-            value: a.title,
+            lifeEvents: 'occupation',
+            eventsDetail1: a.group,
+            eventsDetail2: a.title,
```

Comment:
@ryanctan I noticed you changed the names for the columns to export here; could you explain a bit why you did so? I'm assuming it's for readability's sake, but just want to make sure everyone's on board with this before we merge (as this could make people who work with the data currently have to change their workflows a bit since the column names have changed).

Reply:
Changes were made based on #73

Reply:
@ryanctan I believe the intention (at least the last time @ceserani and I discussed this) was probably for column names in the exported tsv files to have spaces in them. i.e., "Life events" instead of "lifeEvents", "Events Detail 1" instead of "eventsDetail1", etc. That way, the tsv column names are more human-readable. We can probably make another issue for this though.